addons/l10n_mn/data/l10n_mn_chart_data.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+        <menuitem id="account_reports_mn_statements_menu" name="Mongolia" parent="account.menu_finance_reports" sequence="0" groups="account.group_account_user"/>
+
+        <record id="mn_chart_1" model="account.chart.template">
+            <field name="name">Mongolia</field>
+            <field name="bank_account_code_prefix">11</field>
+            <field name="cash_account_code_prefix">10</field>
+            <field name="transfer_account_code_prefix">16</field>
+            <field name="code_digits">8</field>
+            <field name="currency_id" ref="base.MNT"/>
+            <field name="spoken_languages" eval="'lang_mn'"/>
```

Comment:
We don't want multilang, so I don't know what this spoken_languages is doing here

Reply:
Right, I remove it.